BUILD

```diff
@@ -199,9 +199,10 @@ cc_library(
 
 objc_library(
```

Comment:
I can't say I've looked at the build file before, but it seems like this is the same as the target `objectivec`, but building it differently.  I'd suggest making one an alias for the other to avoid breaking folks and using globs is likely better than listing all the files.  for non_arc_srcs, just use a glob that excludes `GPBProtocolBuffers.m`.

Reply:
Thanks for the review -- I've determined an alternative way to fix this in my project so to avoid polluting the BUILD file I'll go ahead and close this PR.